### ISSUE TITLE
Rubocop fails on Rake but not executed directly

### ISSUE THREAD

**Issue by theycallmeswift** [rubocop-hq/rubocop#8974]
## Expected behavior

When I run `bundle exec rubocop`, it passed successfully. 

```
Inspecting 47 files
...............................................

47 files inspected, no offenses detected
```

**Full Log:** https://gist.github.com/theycallmeswift/5cec3a01316739327af90148d6d3ce0e#file-non-rake-log

## Actual behavior

However, when I run it as a rake task (`bundle exec rake rubocop`), I get the following error:

```
47 files inspected, no offenses detected

37 errors occurred:
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/Gemfile:5:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/Gemfile:32:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/Gemfile:38:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/Gemfile:50:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/Gemfile:60:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/Rakefile:14:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/Rakefile:21:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/application.rb:21:4.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/environments/development.rb:4:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/environments/development.rb:20:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/environments/production.rb:6:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/environments/test.rb:3:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/initializers/inline_svg.rb:3:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/initializers/simple_form.rb:11:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/initializers/simple_form.rb:24:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/initializers/wrap_parameters.rb:9:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/config/routes.rb:3:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/db/migrate/20201024175350_create_delayed_jobs.rb:6:4.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/lib/tasks/dev.rake:6:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/lib/tasks/dev.rake:8:4.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/factories.rb:3:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/rails_helper.rb:13:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/rails_helper.rb:22:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/spec_helper.rb:17:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/spec_helper.rb:18:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/spec_helper.rb:22:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/action_mailer.rb:3:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/action_mailer.rb:4:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/chromedriver.rb:5:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/chromedriver.rb:9:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/chromedriver.rb:21:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/chromedriver.rb:22:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/chromedriver.rb:26:2.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/factory_bot.rb:5:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/i18n.rb:3:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/shoulda_matchers.rb:3:0.
An error occurred while Standard/SemanticBlocks cop was inspecting /Users/theycallmeswift/dev/repo/spec/support/shoulda_matchers.rb:4:2.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop-hq/rubocop/issues

Mention the following information in the issue report:
1.0.0 (using Parser 2.7.2.0, rubocop-ast 1.1.0, running on ruby 2.6.6 x86_64-darwin19)
RuboCop failed!
```

Full Log: https://gist.github.com/theycallmeswift/5cec3a01316739327af90148d6d3ce0e#file-rake-log

## Steps to reproduce the problem

I've been able to trace the issue down to the `standard` gem.  It seems like `procedural_method?` is being invoked by rake, but not by calling rubocop directly. I can't seem to sort out **why** this is happening though because the two CLI objects are being invoked with the same options and have the same config files loaded as far as I can tell.

```ruby
    138: def procedural_method?(method_name)
 => 130:   cop_config["ProceduralMethods"].map(&:to_sym).include?(method_name)
    141: end
```

**Source:** https://github.com/testdouble/standard/blob/c9c5e2f41c1939393e259fa532f63bdbf2ea537b/lib/standard/cop/semantic_blocks.rb#L138-L140

```ruby
> cop_config
=> {"Enabled"=>true}
```

## RuboCop version

Include the output of `rubocop -V` or `bundle exec rubocop -V` if using Bundler. Here's an example:

```
$ [bundle exec] rubocop -V
1.0.0 (using Parser 2.7.2.0, rubocop-ast 1.1.0, running on ruby 2.6.6 x86_64-darwin19)
  - rubocop-performance 1.8.1
  - rubocop-rails 2.8.1
```

## Other Notes:

This does **not** appear to be related to #2134

**Comment by theycallmeswift** [comment]
My issue was that standard is not actually compatible with a separate Rubocop implementation. Make sure you're not using them together!
